rules/windows/defense_evasion_masquerading_trusted_directory.toml

```diff
@@ -56,6 +61,18 @@ process where host.os.type == "windows" and event.type == "start" and
         "?:\\Windows\\Downloaded Program Files\\*.exe",
         "?:\\Windows\\Temp\\.opera\\????????????\\CProgram?FilesOpera*\\*.exe",
         "?:\\Windows\\Temp\\.opera\\????????????\\CProgram?Files?(x86)Opera*\\*.exe"
+  ) and
+  not (
+    event.dataset == "crowdstrike.fdr" and
```

Comment:
Just curious, what is this about?

Reply:
To not execute these wildcards unless the dataset is Crowdstrike, so we don't make the query more expensive by adding it